minimisers/DREAM/runDREAM.m

```diff
@@ -93,8 +87,7 @@
 
 
 % Run the sampler....
-Meas_info = struct('Y',0,'N',0);
-[chain,dreamOutput,~] = ratDREAM(DREAMPar,ParInfo,Meas_info,ratInputs);
+[chain,dreamOutput,~] = ratDREAM(DREAMPar,ParInfo,ratInputs);
```

Comment:
Given the other version of this file has been deleted, could you please rename `ratDREAM` as `DREAM`.